aries_cloudagent/ledger/multiple_ledger/indy_manager.py

```diff
@@ -53,7 +54,11 @@ def __init__(
 
     async def get_write_ledger(self) -> Optional[Tuple[str, IndySdkLedger]]:
```

Comment:
It seems like the use of IndySdkLedger instead of BaseLedger in this class is introducing an unnecessary dependency on indy-sdk?

Reply:
Good catch thanks, I'll take a look ...

Reply:
This is the indy sdk manager (as opposed to the indy-vdr manager)

Reply:
(The bug that this PR is addressing is specific to indy-sdk and doesn't occur when using indy-vdr)